plugins/Marketplace/resources/qml/OnboardBanner.qml

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.1
+
+import UM 1.6 as UM
+import Cura 1.6 as Cura
+
+// Onboarding banner.
+Rectangle
+{
+    property bool bannerVisible
```

Comment:
Why duplicate the property here? Visible is already a property of the rectangle.

Reply:
https://github.com/Ultimaker/Cura/pull/10948/commits/f8d9f1e39c98546a31705d4b6fe489427f750810